### PR TITLE
Update 'Installing Ansible on FreeBSD'.

### PR DESCRIPTION
**Pull request by vbotka**
##### SUMMARY

Update the installation on FreeBSD

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by bcoca — APPROVED** [review]

File: docs/docsite/rst/installation_guide/intro_installation.rst [hunk 1]
```diff
@@ -326,36 +326,62 @@ To install the newest version, you may need to unmask the Ansible package prior
 
     $ echo 'app-admin/ansible' >> /etc/portage/package.accept_keywords
 
+
 Installing Ansible on FreeBSD
 -----------------------------
 
-Though Ansible works with both Python 2 and 3 versions, FreeBSD has different packages for each Python version.
-So to install you can use:
+You can install Ansible on FreeBSD either from a package or from a port. See the FreeBSD handbook `Chapter 4. Installing Applications: Packages and Ports <https://docs.freebsd.org/en/books/handbook/ports/>`_. Best practice is to use the packages on the fresh installation of the system and both update and upgrade from the ports later. It's not recommended to mix the installation from packages and ports, i.e. keep updating and upgrading form the ports. See the warning in the FreeBSD handbook `4.5. Using the Ports Collection <https://docs.freebsd.org/en/books/handbook/ports/#ports-using>`_.
```
Comment (Akasurde) [inline docs/docsite/rst/installation_guide/intro_installation.rst]:
```suggestion
You can install Ansible on FreeBSD either from a package or from a port. See the FreeBSD handbook `Chapter 4. Installing Applications: Packages and Ports <https://docs.freebsd.org/en/books/handbook/ports/>`_. A best practice is to use the packages on the fresh installation of the system and both update and upgrade from the ports later. It's not recommended to mix the installation from packages and ports, i.e. keep updating and upgrading from the ports. See the warning in the FreeBSD handbook `4.5. Using the Ports Collection <https://docs.freebsd.org/en/books/handbook/ports/#ports-using>`_.
```

**Review by Akasurde — APPROVED** [review]

File: docs/docsite/rst/installation_guide/intro_installation.rst [hunk 2]
```diff
@@ -326,36 +326,62 @@ To install the newest version, you may need to unmask the Ansible package prior
 
     $ echo 'app-admin/ansible' >> /etc/portage/package.accept_keywords
 
+
 Installing Ansible on FreeBSD
 -----------------------------
 
-Though Ansible works with both Python 2 and 3 versions, FreeBSD has different packages for each Python version.
-So to install you can use:
+You can install Ansible on FreeBSD either from a package or from a port. See the FreeBSD handbook `Chapter 4. Installing Applications: Packages and Ports <https://docs.freebsd.org/en/books/handbook/ports/>`_. A best practice is to use the packages on the fresh installation of the system and both update and upgrade from the ports later. It's not recommended to mix the installation from packages and ports, i.e. keep updating and upgrading from the ports. See the warning in the FreeBSD handbook `4.5. Using the Ports Collection <https://docs.freebsd.org/en/books/handbook/ports/#ports-using>`_.
```
Comment (samccann) [inline docs/docsite/rst/installation_guide/intro_installation.rst]:
```suggestion
You can install Ansible on FreeBSD either from a package or from a port. See the FreeBSD handbook `Chapter 4. Installing Applications: Packages and Ports <https://docs.freebsd.org/en/books/handbook/ports/>`_. A best practice is to use the packages on the fresh installation of the system and both update and upgrade from the ports later. It's not recommended to mix the installation from packages and ports, that is,  keep updating and upgrading from the ports. See the warning in the FreeBSD handbook `4.5. Using the Ports Collection <https://docs.freebsd.org/en/books/handbook/ports/#ports-using>`_.
```

File: docs/docsite/rst/installation_guide/intro_installation.rst [hunk 3]
```diff
@@ -326,36 +326,62 @@ To install the newest version, you may need to unmask the Ansible package prior
...
 
-.. code-block:: bash
 
-    $ sudo pkg install py27-ansible
+FreeBSD packages
+^^^^^^^^^^^^^^^^
 
-or:
+The installation from the packages is simpler, compared to the installation from the ports. See the details in the FreeBSD handbook `4.3. Finding Software <https://docs.freebsd.org/en/books/handbook/ports/#ports-finding-applications>`_ and `4.4. Using pkg for Binary Package Management <https://docs.freebsd.org/en/books/handbook/ports/#pkgng-intro>`_. Take a look at available packages, e.g.
```
Comment (samccann) [inline docs/docsite/rst/installation_guide/intro_installation.rst]:
```suggestion
The installation from the packages is simpler, compared to the installation from the ports. See the details in the FreeBSD handbook `4.3. Finding Software <https://docs.freebsd.org/en/books/handbook/ports/#ports-finding-applications>`_ and `4.4. Using pkg for Binary Package Management <https://docs.freebsd.org/en/books/handbook/ports/#pkgng-intro>`_. Take a look at available packages, for example:
```

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @vbotka for the Ansible docs fix! We talk about Ansible documentation on maxtrix at [#docs:ansible.im](https://matrix.to/#/#docs:ansible.im) and on libera IRC at #ansible-docs if you ever want to join us and chat about the docs!

**Comment by vbotka** [comment]
> ... libera IRC at #ansible-docs if you ever want to join us and chat about the docs!

Thank you! I joined the channel.

**Pull request merged**